PolyEngine/Engine/Src/InputWorldComponent.hpp

```diff
@@ -1,33 +1,37 @@
 #pragma once
 
+#include <initializer_list>
 #include <Vector.hpp>
+#include <EnumUtils.hpp>
 #include "ComponentBase.hpp"
 #include "KeyBindings.hpp"
 
 namespace Poly
 {
-	namespace InputSystem
+	class ENGINE_DLLEXPORT InputWorldComponent : public ComponentBase
 	{
-		class InputWorldComponent : public ComponentBase
+	friend void InputSystem::InputPhase(World*);
+	public:
+		const bool IsPressed(const std::initializer_list<eKey>& list)	
```

Comment:
Alongside this method please add `bool IsPressed(eKey key)` as a convinience. Move the implementation to cpp.
And the const needs to be after method specification. for example: `bool IsPressed(const std::initializer_list<eKey>& list) const;`